documentation/loginAndLogoutFlow.md

```diff
@@ -50,6 +47,9 @@ The user login process is as follows:
    1. The end result of this process is a description of the configuration to be applied to system (described per appplication, filtered according to users privacy settings, etc). The even `onMatchDone` signals that we have the lifecycle instructions ready. For more details on the match maker frameworks internal workings, see: [Match Maker Framework Documentation](MatchMakerFramework.md)
 1. `onMatchDone` is being listened to by the `startLifecycle` (UserLogonStateChange), which applies the settings to the system via the functionality in the LifecycleManager.
 
+**Note**:
+When GPII is deployed for [public work stations use case](https://wiki.gpii.net/w/GPII_Deployment_Structures#Public_work_stations), GPII apps are installed on user machines and request user settings from GPII Cloud. In this deployment, step 1, 2, 3, 6 described above are performed by GPII app and step 4, 5 are performed by GPII Cloud. GPII apps use Resource Owner GPII Token Grant and follow [these steps](https://wiki.gpii.net/w/GPII_OAuth_2_Guide#Resource_Owner_GPII_Token_Grant) to request user settings from GPII Cloud.
```

Comment:
Super minor, but I would pluralize "steps": "In this deployment, steps 1, 2, 3, 6 described above are performed by GPII app and steps 4, 5 are performed by GPII Cloud."